src/diffusers/pipelines/sana_video/pipeline_sana_video_i2v.py

```diff
@@ -377,10 +376,10 @@ def encode_prompt(
 
         bs_embed, seq_len, _ = prompt_embeds.shape
         # duplicate text embeddings and attention mask for each generation per prompt, using mps friendly method
-        prompt_embeds = prompt_embeds.repeat(1, num_videos_per_prompt, 1)
-        prompt_embeds = prompt_embeds.view(bs_embed * num_videos_per_prompt, seq_len, -1)
+        prompt_embeds = prompt_embeds.repeat(1, num_images_per_prompt, 1)
+        prompt_embeds = prompt_embeds.view(bs_embed * num_images_per_prompt, seq_len, -1)
```

Comment:
```suggestion
        prompt_embeds = prompt_embeds.repeat(1, num_videos_per_prompt, 1)
        prompt_embeds = prompt_embeds.view(bs_embed * num_videos_per_prompt, seq_len, -1)
```
Can you revert this to using `num_videos_per_prompt`, as this is a video pipeline and so `encode_prompt` should use a `num_videos_per_prompt` rather than a `num_images_per_prompt` argument? Sorry for the confusion!